<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InventoryWindow</class>
 <widget class="QWidget" name="InventoryWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Georgia</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Inventory</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="unlockedEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>351</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="drop">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>340</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Drop</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>340</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="use">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>340</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Use</string>
   </property>
  </widget>
  <widget class="QPushButton" name="apply">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>340</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply Boost/Potion</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Georgia</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Collected Items:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="lockedEdit">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>50</y>
     <width>351</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>20</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Georgia</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Items:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
